packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -332,8 +332,10 @@ export class VSXExtensionComponent extends AbstractVSXExtensionComponent {
                         <span className='name'>{displayName}</span> <span className='version'>{version}</span>
                     </div>
                     <div className='stat'>
-                        {downloadCount && <span className='download-count'><i className='fa fa-download' />{downloadCompactFormatter.format(downloadCount)}</span>}
-                        {averageRating && <span className='average-rating'><i className='fa fa-star' />{averageRating.toFixed(1)}</span>}
+                        {downloadCount && downloadCount !== 0 ?
+                            <span className='download-count'><i className='fa fa-download' />{downloadCompactFormatter.format(downloadCount)}</span> : ''}
+                        {averageRating && averageRating.toFixed(1) !== '0.0' ?
```

Comment:
`averageRating.toFixed(1) !== '0.0'` it looks strange, probably `!!averageRating` also will be enough?